app/src/test/java/com/duckduckgo/app/browser/BrowserViewModelTest.kt

```diff
@@ -81,19 +85,26 @@ class BrowserViewModelTest {
 
     @Mock private lateinit var showOnAppLaunchOptionHandler: ShowOnAppLaunchOptionHandler
 
+    @Mock private lateinit var userStageStore: UserStageStore
+
     private val fakeShowOnAppLaunchFeatureToggle = FakeFeatureToggleFactory.create(ShowOnAppLaunchFeature::class.java)
 
     private lateinit var testee: BrowserViewModel
 
     private val skipUrlConversionOnNewTabFeature = FakeFeatureToggleFactory.create(SkipUrlConversionOnNewTabFeature::class.java)
 
+    private val swipingTabsFeature = FakeFeatureToggleFactory.create(SwipingTabsFeature::class.java)
+
+    private val swipingTabsFeatureProvider = SwipingTabsFeatureProvider(swipingTabsFeature)
+
     @Before
     fun before() {
         MockitoAnnotations.openMocks(this)
 
         doReturn(MutableLiveData<AppEnjoymentPromptOptions>()).whenever(mockAppEnjoymentPromptEmitter).promptType
 
         configureSkipUrlConversionInNewTabState(enabled = true)
+        swipingTabsFeature.self().setRawStoredState(State(enable = true))
```

Comment:
Should we have tests with this feature disabled?
`swipingTabsFeature.self().setRawStoredState(State(enable = false))`

Reply:
[I set the flag to false by default](https://github.com/duckduckgo/Android/pull/5408/commits/0441e83609423958574d1b01d75aea628d0dd579) and only enable it for new tests.